.github/workflows/ci-pr.yml

```diff
@@ -107,3 +107,38 @@ jobs:
     secrets:
       SMITHY_RS_PULL_REQUEST_CDN_S3_BUCKET_NAME: ${{ secrets.SMITHY_RS_PULL_REQUEST_CDN_S3_BUCKET_NAME }}
       SMITHY_RS_PULL_REQUEST_CDN_ROLE_ARN: ${{ secrets.SMITHY_RS_PULL_REQUEST_CDN_ROLE_ARN }}
+
+  semver-checks:
+    name: check the semver status of this PR
+    runs-on: smithy_ubuntu-latest_8-core
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        path: smithy-rs
+        ref: ${{ inputs.git_ref }}
+    - name: Get PR info
+      id: check-breaking-label
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const response = await github.rest.pulls.get({
+            pull_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+          });
+          const labels = response.data.labels.map(l => l.name);
+          const isBreaking = labels.includes("breaking-change");
+          const data = {
+            labels,
+            isBreaking
+          };
+          console.log("data:", data);
```

Comment:
Left in for debug purposes I presume?

Reply:
yeah, just to make it easier to see what's going on